deployment/ansible/roles/raster-foundry.app/tasks/static-files.yml

```diff
@@ -41,7 +41,7 @@
 
 - name: Run Django collectstatic command
   django_manage: command=collectstatic app_path="{{ django_home }}"
-  environment: django_config
+  environment: "{{ envdir_config | combine(django_config) }}"
```

Comment:
Note that I also created a helper method called `is_truthy` that checks for nulls so this is not entirely necessary (but probably a good idea anyway).